components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -386,7 +381,11 @@ function StarcraftMatchGroupInput.SubMatchStructure(match)
 				winner = 0
 			}
 			--adjust sub-match scores
-			if tonumber(match['map' .. i].winner) == 1 then
+			if match['map' .. i].map and String.startsWith(match['map' .. i].map, 'Submatch') then
```

Comment:
`match['map' .. i]` should be extracted to a local variable
(Separate PR)

Reply:
if necessary will do it in this pr, but since i am working on a general clean up of this module i would like to do it via that clean up instead

https://liquipedia.net/commons/Module:MatchGroup/Input/Starcraft/cleanup

Reply:
As I wrote in the comment, keep it to a separate PR